### PR TITLE
UT-3483 Changed label name for strip custom namespace option.

### PR DESCRIPTION
**Pull request by nat-w**
Label is now more clear for the strip custom namespace option.

### CODE REVIEW

**Comment by nat-w** [comment]
Screen Shot 2020-07-15 at 11 51 34 AM

**Comment by nat-w** [comment]
Should I update the docs with the new label and picture too?

File: Integrations/Autodesk/maya/scripts/unityCommands.mel [hunk 1]
```diff
@@ -319,6 +325,7 @@ proc setExportSetAttributes(
     global string $UnityFbxAnimFileNameAttr;
     global string $UnityFbxStripNamespaceAttr;
     global string $UnityFbxNamespaceAttr;
+	global string $UnityFbxNamespaceAttrNiceName;
```
Comment (lassond) [inline Integrations/Autodesk/maya/scripts/unityCommands.mel:328]: Nitpick: The indent is not correct here

**Comment by lassond** [comment]
Good job! 
The UX department approves this change!

Please push a commit with the indent fix and then merge it

**Pull request merged**